.gitlab-ci.yml

```diff
@@ -25,14 +27,31 @@ sniffers:compose:
   - composer -vvv -V
   - time composer validate --profile --strict
 
+sniffers:front:
+  cache:
+    key: "front-packages"
```

Comment:
we need to resolve case when 2 tasks in even different project use that cache, we can add more specific key cache. (per project/per commit/per story) or whatever.

Reply:
in near future, CI jobs will run accross multiple servers
so better make cache accessible from elsewhere

Reply:
Moreover I'm not sure that this key is needed

Reply:
Follow up issue created in contrib board